### PR TITLE
Allows xenos with more than 300 hours to have a double letter postfix

### PR DESCRIPTION
**Pull request by usnpeepoo**
## About The Pull Request

Xenos with 300+ hours can now use another letter for their postfix, instead of a number.
"To clarify: Previously you could also use 2 characters as the postfix but the last of them had to be a number, This allows you to have two letters.

Allowed now:

BR-XXX-AD

XE-XXX-NO

Not Allowed:

HUM-XXX-AN

HU-XXX-3N

etc."
All credit to Nivrak, this is a port of their original MR which was architect/Nanu approved 6 months ago. You can find the link here. I've edited it to be 300 and not 500 as well.
https://gitlab.com/cmdevs/colonial-warfare/-/merge_requests/2623

## Why It's Good For The Game

A better reward for name tags, as well as letting xenos be able to have more freedom in their name choice. This change unlocks tags previously locked by being forced to come up with a number similar to it. It also helps with rule breaking tags being more clear if a player chooses to use the extra letter. QOL basically.

## Changelog
:cl:Usnpeepoo
add: Xenos with more than 300 hours of playtime (as xeno) can now use a double letter postfix.
/:cl:

### CODE REVIEW

**Review by Shad0vvs — CHANGES_REQUESTED** [review]
Sent some things in discord.

**Comment by PhantomEpicness** [comment]
Xenos with 500 hours should have the option of switching the numbers and letters

like
12-XENO-45

File: code/modules/client/preferences.dm [hunk 1]
```diff
@@ -1248,7 +1248,8 @@ var/const/MAX_SAVE_SLOTS = 10
 							switch(ascii_char)
 								// A  .. Z
 								if(65 to 90)			//Uppercase Letters will work on first char
-									if(!first_char)
+									if(!first_char && playtime < 300 HOURS)
+										to_chat(user, SPAN_WARNING(FONT_SIZE_BIG("You need to play [time_left_until(300 HOURS, playtime, 1 HOURS)] more hours to unlock double letter xeno postfix.")))
```
Comment (stanalbatross) [inline code/modules/client/preferences.dm:1252]: needs to check for the length being TWO or this will let people have unlimited first prefixes

**Review by stanalbatross — CHANGES_REQUESTED** [review]

**Review by Shad0vvs — APPROVED** [review]

**Comment by usnpeepoo** [comment]
> Your Fix/Feature pull request is currently below zero (-5). Maintainers may close future Feature/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

FUCK YOU!!!!

**Pull request merged**